compose.yml

```diff
@@ -30,5 +30,11 @@ services:
     ports:
       - "127.0.0.1:8580:8080"
 
+  tmt-osaamissuosittelija-api:
```

Comment:
Maybe this should use a profile so that it would be possible to run the app locally without the external private dependency.